src/coprocessor/dag/rpn_expr/mod.rs

```diff
@@ -207,8 +210,8 @@ fn map_pb_sig_to_rpn_func(value: ScalarFuncSig, children: &[Expr]) -> Result<Rpn
         ScalarFuncSig::InDuration => compare_in_fn_meta::<Duration>(),
         ScalarFuncSig::InJson => compare_in_fn_meta::<Json>(),
         _ => return Err(box_err!(
-            "ScalarFunction {:?} is not supported in batch mode",
-            value
+                "ScalarFunction {:?} is not supported in batch mode",
```

Comment:
Revert this change.

Reply:
my `vim` formatted my code despite the `#[rustfmt::skip]` :laughing:

Reply:
You can undo the indent.